tasks/libs/releasing/json.py

```diff
@@ -23,6 +23,24 @@
 # The order matters, eg. when fetching matching tags for an Agent 6 entry,
 # tags starting with 6 will be preferred to tags starting with 7.
 COMPATIBLE_MAJOR_VERSIONS = {6: ["6", "7"], 7: ["7"]}
+RELEASE_JSON_FIELDS_TO_UPDATE = [
+    "INTEGRATIONS_CORE_VERSION",
+    "OMNIBUS_SOFTWARE_VERSION",
+    "OMNIBUS_RUBY_VERSION",
+    "MACOS_BUILD_VERSION",
+]
+
+UNFREEZE_REPO_AGENT = "datadog-agent"
+INTERNAL_DEPS_REPOS = ["omnibus-software", "omnibus-ruby", "datadog-agent-macos-build"]
+DEPENDENT_REPOS = INTERNAL_DEPS_REPOS + ["integrations-core"]
+ALL_REPOS = DEPENDENT_REPOS + [UNFREEZE_REPO_AGENT]
+UNFREEZE_REPOS = INTERNAL_DEPS_REPOS + [UNFREEZE_REPO_AGENT]
+DEFAULT_BRANCH = {
```

Comment:
nit - I think it would better reflect this is a dict

```suggestion
DEFAULT_BRANCHES = {
```